### PR TITLE
[E2E] Adding script for generating index.ts file.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
The file index.ts contains all files that are exported from the project. These files then can be imported in other project and simply extended/overwritten. This will automate the process of addition files to index.ts.
There are folders defined from which the typescript files should be written to the index.ts. There is also list for excluded files, e.g. CheReporter. 
The index.ts file is generated each time the test is run. The execution time of generation of index.ts is quite short [1] so it should not negatively affect user experience when developing new tests.
Screenshot from 2019-08-19 13-16-48

### CODE REVIEW

**Comment by Katka92** [comment]
crw-ci-test

File: e2e/generateIndex.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+echo "Generating index.ts file..."
+echo "import * as inversifyConfig from './inversify.config';
+export { inversifyConfig };
+export * from './inversify.types';
+export * from './TestConstants';
+" > index.ts
+
+listOfDirs="driver utils pageobjects"
+listOfExcludes="./driver/CheReporter.ts"
+for dir in $listOfDirs
```
Comment (Ohrimenko1988) [inline e2e/generateIndex.sh:12]: Does this script work with nested folders? For example "pageobjects" -> "dashboard"

Comment (Katka92) [reply e2e/generateIndex.sh:12]: In short yes - there is already index.ts file generated, you can take a look https://github.com/eclipse/che/pull/14281/files#diff-14f7a35f8fc13dbe57f6822026fa0f1dR12

**Review by Ohrimenko1988 — APPROVED** [review]

**Comment by Katka92** [comment]
crw-ci-test

**Review by rhopp — APPROVED** [review]

**Pull request merged**